tools/dataset/labelbee_to_coco.py

```diff
@@ -0,0 +1,154 @@
+                        "coco_url":"http://creativecommons.org/licenses/by-nc-sa/2.0/",
+                        "height":current_json_content["height"],
+                        "width":current_json_content["width"],
+                        "date_captured":"2022/03/24",
+                        "flickr_url":"http://creativecommons.org/licenses/by-nc-sa/2.0/",
+                        "id": i + 1
+                        }
+            anno_info["images"].append(img_anno)
+
+            for j in range(len(current_json_content["step_1"]["result"])):
+                current_bbox = current_json_content["step_1"]["result"][j]
+
+                # 凭借这个id在后面的关键点列表里寻找到对应框内的关键点
+                bbox_id = current_bbox["id"]
+                keypoints_list = []
+                keypoints_num = 0
+                area = int(current_bbox["width"]) * int(current_bbox["height"])
+
+                current_json_content["step_2"]["result"]
+                for k in range(len(current_json_content["step_2"]["result"])):
+                    if current_json_content["step_2"]["result"][k]["sourceID"] == bbox_id:
+                        current_bbox_keypoints = current_json_content["step_2"]["result"][k:k+17]
+                        break
+
+                for k in range(len(current_bbox_keypoints)):
+                    if current_bbox_keypoints[k]["attribute"] == "1":
+                        # 这个点 可视
+                        keypoints_num = keypoints_num + 1
+                        keypoints_list.append(int(current_bbox_keypoints[k]["x"]))
+                        keypoints_list.append(int(current_bbox_keypoints[k]["y"]))
+                        keypoints_list.append(2)
+                    else:
+                        # 这个点 不可视
+                        keypoints_list.append(0)
+                        keypoints_list.append(0)
+                        keypoints_list.append(0)
+
+                keypoints_anno = {
+                                    "segmentation":[[]],
+                                    "num_keypoints":keypoints_num,
+                                    "area":area,
+                                    "iscrowd":0,
+                                    "keypoints":keypoints_list,
+                                    "image_id": i + 1,
+                                    "bbox":[int(current_bbox["x"]), int(current_bbox["y"]),
+                                            int(current_bbox["width"]), int(current_bbox["height"])],
+                                    "category_id":1,
+                                    "id": l
+                                  }
+                l = l + 1
+                anno_info["annotations"].append(keypoints_anno)
+
+        with open(args.output_file + data_set[n] +  '.json', 'w') as f:
+            f.write(json.dumps(anno_info, ensure_ascii=False, indent=4, separators=(',', ':')))
+            f.close()
+
+        print(str(data_set[n]) + "标签转化完成！！！！")
+
+
+if __name__=='__main__':
+    main()
```

Comment:
This PR has some linting problems. Please follow
https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md to setup pre-commit.